### PR TITLE
[Files module]Improve remote functions and Update display annotations

### PR DESCRIPTION
**Pull request by SanduDS**
## Approach
1.  Added new operation to upload byte arrays
2.  Added a new test case
3. Recorrect documentation typos.
4. Update Display annotations of the file module
5. Removed warning messages for isolated functions

## Automation tests
 - Unit tests 
   > Done
 - Integration tests
   > Done

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: storageservice/modules/files/file_client.bal [hunk 1]
```diff
@@ -253,12 +250,12 @@ public client class FileClient {
...
     # + azureDirectoryPath - Path of the azure directory
     # + return - If success, returns true.  Else returns error
-    @display {label: "Write content to file"}
-    remote function putRange(@display {label: "File share name"} string fileShareName,
-                             @display {label: "Local file path"} string localFilePath, 
-                             @display {label: "Name of the file in azure"} string azureFileName, 
-                             @display {label: "Azure directory path"} string? azureDirectoryPath = ()) 
-                             returns @tainted @display {label: "Status"} boolean|error {
+    @display {label: "Write Content To Azure File"}
+    isolated remote function putRange(@display {label: "File Share Same"} string fileShareName,
```
Comment (abeykoon) [inline storageservice/modules/files/file_client.bal:254]: File Share Name? However, what is meant by `File Share Name`?

Comment (abeykoon) [reply storageservice/modules/files/file_client.bal:254]: There is a few places we have done same mistake.

File: storageservice/modules/files/file_client.bal [hunk 2]
```diff
@@ -430,34 +425,45 @@ public client class FileClient {
...
+
+    # Upload a byte array directly to the fileshare.
+    # 
+    # + fileShareName - Name of the fileShare
+    # + fileContent - File content as a byte array
+    # + azureFileName - Name of the file in Azure
+    # + azureDirectoryPath - Directory path in Azure
+    # + return - If success, returns true.  Else returns error
+    isolated remote function directUploadFileAsByteArray(@display {label: "File Share Name"} string fileShareName, 
+                                 @display {label: "File Content Array"} byte[] fileContent,
```
Comment (abeykoon) [inline storageservice/modules/files/file_client.bal:460]: `File Content (Byte Array)` - however, not useful for Choreo at all.

File: storageservice/modules/files/file_client.bal (hunk 1 above)
Comment (SanduDS) [reply storageservice/modules/files/file_client.bal]: File Shares are the file containers of Azure File Service.

File: storageservice/modules/files/file_client.bal (hunk 2 above)
Comment (SanduDS) [reply storageservice/modules/files/file_client.bal]: Updated the Display annotations. It is useful to the user when needs to upload files that get as byte array such as Gmail attachments etc.

Comment (abeykoon) [reply storageservice/modules/files/file_client.bal]: +1. Yes, we can get it as an expression variable.

File: storageservice/modules/files/file_client.bal (hunk 1 above)
Comment (abeykoon) [reply storageservice/modules/files/file_client.bal]: +1

**Review by abeykoon — APPROVED** [review]

**Pull request merged**